packages/backend/src/managers/playground/McpServerManager.ts

```diff
@@ -0,0 +1,75 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import type { Disposable } from '@podman-desktop/api';
+import fs from 'node:fs';
+import path from 'node:path';
+
+const MCP_SETTINGS = 'mcp-settings.json';
+
+export enum McpServerType {
+  STDIO = 'stdio',
+  SSE = 'sse',
+}
+
+export interface McpServer {
+  name: string;
+  enabled: boolean;
+  type: McpServerType;
+  command: string;
+  args: string[];
+  url: string;
+  headers: Record<string, string>;
+}
+
+// TODO: Consider using JsonWatcher in case the file is going to be updated externally
```

Comment:
Should we consider this as a potential use case? Like the user would be doing the update until he found something that he want?